src/index.ts

```diff
@@ -247,7 +247,7 @@ class BuxClient implements TransportService {
   /**
    * Admin only: Get a paymail by address
    *
-   * @param address string Paymail address (ie. example@bux.org)
+   * @param address string Paymail address (i.e. example@spv-wallet.org)
```

Comment:
I racall that we established that a domain should be `example.com` e.g "alias@example.com"